script/fetch-recorded-data

```diff
@@ -189,10 +193,28 @@ async function main() {
       `,
       [caseRecords]
     )
+    console.log(`Saved ${caseRecords.length} records to case_data table...`)
     await db.query(
       'RENAME TABLE case_data TO case_data_old, case_data_import TO case_data'
     )
+  } catch (e) {
+    console.error('Failed to insert case data')
+    console.error(e)
+    isError = true
+  } finally {
+    await db.query('DROP TABLE IF EXISTS case_data_old')
+    await db.query('DROP TABLE IF EXISTS case_data_import')
+  }
+
+  try {
+    console.log('Inserting interventions data')
+    if (!usInterventionRecords.length) {
+      throw new Error('No interventions found')
+    }
+    await db.query('COMMIT')
```

Comment:
I think this `COMMIT` should be earlier, at the end of the previous `try` after you have renamed the tables.